fc/cli/main.py

```diff
@@ -14,6 +14,7 @@ def cli():
 
 cli.add_command(triage.triage)
 cli.add_command(backlog.backlog)
+backlog.backlog.add_command(backlog.create)
```

Comment:
This line should be able to be removed with the above suggestion of
`@click.group` -> `@backlog.group`.